### ISSUE TITLE
Delete files when no longer provided

### ISSUE THREAD

**Issue by bbaugher** [cerner/cerner_tomcat#3]

**Comment by EdwardSkoviak** [comment]
I'm assuming the idea here was we'd delete the remote_files/templates/cookbook_files installed by the sub-resources in previous runs? I don't really think theres anything we can do to support that, unless we are to store the nodes previous configuration to file. We could also create a re-install action which deletes everything and deploys a fresh installation.

**Comment by bbaugher** [comment]
Yes the idea was if someone no longer provided a webapp or a file landed in tomcat's lib directory it should be cleaned up. For webapp's its pretty easy since its delete anything not defined/configured. For random files though its harder

**Comment by bbaugher** [comment]
This probably isn't super important
